crypto/keyring/testdata/keys/keys.db/CURRENT

```diff
@@ -0,0 +1 @@
+MANIFEST-000004
```

Comment:
can you remove these files or add them to the gitignore? 🙏

Reply:
`.gitignore` 👌

Reply:
They are test fixtures that I've generated with the previous version of gaia. They live in `testdata` as per `go help test`'s output